<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_exit">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>得意黑</family>
     <pointsize>22</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>831</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>得意黑</family>
        <pointsize>22</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>清理</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_dism">
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Dism++</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_ccleaner">
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>CCleaner</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_spacesniffer">
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>spacesniffer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>得意黑</family>
        <pointsize>22</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_geek">
      <property name="font">
       <font>
        <family>得意黑</family>
        <pointsize>22</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>geek</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
